### PR TITLE
1634780: typo in syspurpose man page

### PR DESCRIPTION
**Pull request by tlhmerry9**
Change foo to usage and quote the commands

### CODE REVIEW

File: syspurpose/man/syspurpose.8 [hunk 1]
```diff
@@ -54,20 +54,20 @@ Show the current system purpose properties
...
-Add a value to a specific list property. E.g. \fBsyspurpose add addons foo
-bar\fP is identical to \fBsyspurpose add-addons foo bar\fP.
+Add a value to a specific list property. E.g. \fB"syspurpose add addons foo
+bar"\fP is identical to \fB"syspurpose add-addons foo bar"\fP.
 .TP
 \fBsyspurpose remove \fIPROPERTY\fP \fIVALUE\fP\fP
-Clear a value from a specific list property. E.g. \fBsyspurpose remove addons
-foo bar\fP is identical to \fBsyspurpose remove-addons foo bar\fP.
+Clear a value from a specific list property. E.g. \fB"syspurpose remove addons
+foo bar"\fP is identical to \fB'syspurpose remove-addons foo bar"\fP.
```
Comment (jirihnidek) [inline syspurpose/man/syspurpose.8:70]: Opening simple quotation marks are ended with double quotation marks.

**Review by jirihnidek — COMMENTED** [review]
Thanks for the PR. I have one small request (comment in the code). When this request will be addressed, then please squash commits into one commit.

**Comment by tlhmerry9** [comment]
I have not squashed a commit before - I ran git rebase -i HEAD~3 (a command I found online) and it took me into an editor.  I wasn't sure what I needed to do while I was in there do I did a git rebase --abort

**Comment by jirihnidek** [comment]
I recommend to learn how to use ViM editor. You never know where you will have to use this unusual editor. Actually it is very powerful visual editor, when you have to use terminal.

You can modify your preference and change default editor to something more user friendly (e.g. gedit). Look at this:

https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables

```bash
export GIT_EDITOR=gedit
export EDITOR=gedit
```

You can added previous line(s) to your `~/.bashrc` file.

I squash commits in PR using "git reset --soft ...". In this case it would be following:

```bash
git reset --soft HEAD~3
# Optionally you can run: 'git log'  to see commit hashes and commit logs
git commit --reedit-message=f854ff0c42e8b5a259fb2ada073324d024006120
# ... You can change the commit message now in your favorite text editor
git push --force origin tmerry/ENT-968
```

**Comment by tlhmerry9** [comment]
Done

**Review by jirihnidek — APPROVED** [review]
ACK

**Pull request merged**